src/service/persistence.js

```diff
@@ -141,10 +141,12 @@ function convertEmail(email) {
         return systemUser
     }
 
-    return email.split('@')[0]
+    //TODO: justice.gov emails no longer match 1:1 with jura usernames, so this may fail post migration to justice.gov
```

Comment:
maybe something like this, it should probably be in a separate pull request though
```suggestion
    // TODO: justice.gov emails no longer match 1:1 with Jira usernames, so this may fail post migration to justice.gov.uk, this hack mostly works but if there are multiple people with the same name this could cause issues.
```